src/DynamoCore/BuiltInAndOperators/BuiltIn.xml

```diff
@@ -144,14 +147,16 @@
     </member>
     <member name="List.TrueForAll">
       <summary>Returns true if all items in the list evaluate to true with the given predicate.</summary>
-      <param name="list">list of values</param>
-      <param name="predicate"></param>
+      <param name="list">List from which to evaluate items by query function</param>
+      <param name="queryFunction">Function to evaluate list</param>
+      <returns name="bool">Returns true for all of the items in the list compared to the query function</returns>
```

Comment:
NIT: `Returns true for all of the items` kind of gives the idea this would return one value of `true` for each item. Also I don't think the word `compare` fits well here because items are not really compared to the `queryFunction` argument, but rather they are evaluated with it. Maybe it would be better to just repeat the text from the summary?

Reply:
Addressed in 2a16aa5